### ISSUE TITLE
Shows, nvim/plugged/nvim-lspconfig/lua/lspconfig/configs.lua:189: bad argument #1 to 'ipairs' (table expected, got nil)

### ISSUE THREAD

**Issue by arionrefat** [williamboman/nvim-lsp-installer#234]
![image](https://user-images.githubusercontent.com/37513070/139936084-0d2b532e-42fa-45e1-b27d-e7383f7e8a8b.png)

**Comment by williamboman** [comment]
Hello! Which servers have you installed?

**Comment by williamboman** [comment]
I think `fix-empty-filetype-attach` (#237) should fix this. I'm assuming what's happening here is that you have one or more server configs with no `filetypes` associated

**Comment by treatybreaker** [comment]
The fix applied in (#237) worked for me after pointing packer to the branch. 

It was an issue with a missing association from the looks of it as you assumed. Seems the lsp server "efm" is missing an association -- removing it resolved the above problem without the patch applied.

**Comment by fernandofleury** [comment]
I'm running into the same issue (but at line 234) if I try to setup the efm server manually:

```lua
local prettier = {
  formatCommand = 'prettierd "${INPUT}"',
  formatStdin = true,
  env = {
    string.format('PRETTIERD_DEFAULT_CONFIG=%s', vim.fn.expand('~/.config/nvim/utils/linter-config/.prettierrc.json')),
  },
}

local languages = {
    typescript = { prettier },
    javascript = { prettier },
    typescriptreact = { prettier },
    javascriptreact = { prettier }
}

require'lspconfig'.efm.setup {
    init_options = {documentFormatting = true},
    filetypes = vim.tbl_keys(languages),
    settings = {
        rootMarkers = {".git/"},
        languages = languages
    }
}
```
